src/Service.GraphQLBuilder/Sql/SchemaConverter.cs

```diff
@@ -243,6 +243,7 @@ public static string GetGraphQLTypeForColumnType(Type type)
                 "Decimal" => DECIMAL_TYPE,
                 "Boolean" => BOOLEAN_TYPE,
                 "DateTime" => DATETIME_TYPE,
+                "DateTimeOffset" => DATETIME_TYPE,
```

Comment:
in my storedprocedure param valuetype pr i needed to change this to have a dedicated datatimeoffset_type to handle differences between DataTime and DateTimeOffset.

Reply:
I didn't end up changing this.